### PR TITLE
Rebuild with python 3.7 and from CDAT (not UV-CDAT) repo

### PR DESCRIPTION
**Pull request by xylar**
Rebuild for Python 3.7, GCC 7, R 3.5.1, openBLAS 0.3.2

closes #8

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
This will not succeed until https://github.com/conda-forge/genutil-feedstock/pull/8 has been merged but I wanted to have it waiting in the wings.

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
Great, I think this one is also working. @dnadeau4 or @doutriaux1, could you please merge this once CI has finished?  Thanks a bunch!

**Comment by doutriaux1** [comment]
@xylar @dnadeau4 cdutil is now noarch let's tag 8.1 on repo and update recipe here to reflect this!

**Comment by xylar** [comment]
@doutriaux1, okay, great!  I'll update later today.

**Comment by xylar** [comment]
Does the same apply for `genutil`?

**Comment by dnadeau4** [comment]
@xylar when travis comes back green I will merge your PR.  this way you will have python 3.7 avail until the  version 8.1 is ready.  It will supersede this version.

I had never seen that script before:
```
   script: "{{ PYTHON }} -m pip install . --no-deps -vv" 
```

Is this gonna run our setup.py script.  We usually try not to use "pip" but setuptools.

**Comment by xylar** [comment]
@dnadeau4, I'm following the instructions here:
https://conda-forge.org/docs/meta.html#use-pip

File: recipe/meta.yaml [hunk 1]
```diff
@@ -6,21 +6,22 @@ package:
...
-  script: python setup.py install
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
-  build:
-    - python
+  host:
+    - python {{python}}
     - setuptools
     - cdat_info
```
Comment (xylar) [inline recipe/meta.yaml:21]: We could probably get rid of `setuptools` and `cdat_info` as requirements, but I haven't tried that myself.

**Comment by xylar** [comment]
@dnadeau4, I'm happy to revert my change to the script line if you prefer to keep it as it is, but I think it's best to follow conda-forge norms where possible.

File: recipe/meta.yaml (hunk 1 above)
Comment (dnadeau4) [reply recipe/meta.yaml:21]: We need setuptools and cdat_info for `cdutil`
https://github.com/CDAT/cdutil/blob/master/setup.py
https://github.com/CDAT/cdutil/blob/master/cdutil/__init__.py

**Comment by dnadeau4** [comment]
No need, conda-forge keeps changing and I just never saw this before.  If it works, than I am good with it.   We created these packages on conda-forge 2 years ago, and it is hard to keep up with them on every details.  

They used to use "travis" only....   We convert to `circleci` before them due to the lack of OSX resources on travis, but we won't do microsoft azure.  ;-)

**Comment by xylar** [comment]
I totally agree, hard to keep up.

**Comment by doutriaux1** [comment]
guys I'm taking care of this don't worry.

**Pull request closed without merging**